modules/nf-core/islandpath/tests/main.nf.test

```diff
@@ -0,0 +1,47 @@
+// nf-core modules test islandpath
+nextflow_process {
+
+    name "Test Process ISLANDPATH"
+    script "../main.nf"
+    process "ISLANDPATH"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gunzip"
+    tag "islandpath"
+
+    test("bacteroides_fragilis - gbff") {
+
+        setup {
+            run("GUNZIP") {
+                script "../../gunzip/main.nf"
+                process {
+                    """
+                    input[0] = Channel.fromList([
+                                    tuple([ id:'test' ], // meta map
+                                    file(params.test_data['bacteroides_fragilis']['genome']['genome_gbff_gz'], checkIfExists: true)),
+                                ])
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = GUNZIP.out.gunzip
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out[0][0]).match() },
```

Comment:
{ assert snapshot(process.out.versions).match() } better?

Reply:
Done in f6d27ca4212d60555e1ca319d2a78df27253d017, thanks